modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+.ROSA `ocm-role` and policy file
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-OCM-Role`
+|An IAM role that enables you to create and maintain ROSA clusters in  {cluster-manager}.
+
+|===
+
+.`sts_ocm_role_trust_policy.json`
+[%collapsible]
+====
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::710019948333:role/RH-Managed-OpenShift-Installer"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "<OCM_account_ID>"
+                }
+            }
+        }
+    ]
+}
+----
+====
+
+.ROSA user role and policy file
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-User-<OCM_user>-Role`
+|An IAM role used by Red{nbsp}Hat to verify the customer's AWS identity.
+
+|===
+
+.`sts_user_role_trust_policy.json`
```

Comment:
I'm not seeing any major dfferences between this confiuration and table 7. Is this intended?

Reply:
Yes :-)